wire/modules/Fieldtype/FieldtypeOptions/FieldtypeOptions.module

```diff
@@ -207,6 +207,10 @@ class FieldtypeOptions extends FieldtypeMulti implements Module {
 	 *
 	 */
 	public function ___markupValue(Page $page, Field $field, $value = null, $property = '') {
+		$interfaces = class_implements($field->inputfieldClass);
+		if(!in_array(InputfieldHasArrayValue::class, $interfaces) && $value instanceof WireArray) {
```

Comment:
I was worried about `class_implements`, but that's actually 5.1+. Didn't think of `::class`. Any better idea than hardcoding the string?